go/buildhelper/helper.go

```diff
@@ -0,0 +1,57 @@
+package buildhelper
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/obscuronet/obscuro-playground/go/buildhelper/buildconstants"
+)
+
+func Addr1PK() *ecdsa.PrivateKey {
+	privateKey, _ := crypto.HexToECDSA(buildconstants.ADDR1PK)
+	return privateKey
+}
+
+func Addr2PK() *ecdsa.PrivateKey {
+	privateKey, _ := crypto.HexToECDSA(buildconstants.ADDR2PK)
+	return privateKey
+}
+
+func Addr3PK() *ecdsa.PrivateKey {
```

Comment:
the existence of these functions highlights that we need another concept.
The "EthereumWallet", which has to be able to sign with a key that controls ether.
```
type EthereumWallet interface{
   sign(TxData) (*Transaction, error)
}
```
One implementation should be with a key provided in the constructor, and another might be with an keystore file, another with an hsm, etc.

This "EthereumWallet" needs to be a field in the main host node type.

Reply:
Yep, that's a good way of layering it 👍